src/components/ReserveCars/ReserveCarForm.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { format } from 'date-fns';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import FormComponent from '../Form/FormComponent';
+import { ReserveCarSchema, reserveCarInitialValues } from '../../models/reserveCar.model';
+import { DateField, SelectField, TextInputField } from '../Form/FormField';
+import FormSubmitButton from '../Button/FormSubmitButton';
+import { postReserveCar } from '../../redux/thunk';
+import { MY_RESERVATIONS, USERS_DASHBOARD } from '../../routes/routeConstants';
+import { getNewReservationName } from '../../redux/reservations/reservationSlice';
+
+const ReserveCarFrom = ({ id, username, name }) => {
```

Comment:
- [ ] I was able to reserve in the past:

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/c745dbc4-e7d0-4dab-9bff-390b61c73ef5)

Kindly, add some validation so your users are able only to reserve in the future. 👍